strings/hw_strings.py

```diff
@@ -0,0 +1,32 @@
+import os, string, re
+
+#1)Из текстового файла удалить все слова, содержащие от трех до пяти символов, но при
+#этом из каждой строки должно быть удалено только четное количество таких слов.
+
+def remove_words_from_file(filename):
+    '''Function removes from the input file even number of the words containing from 3 to 5 characters.'''
+    
+    if not os.path.exists(filename):
+        raise FileNotFoundError('File does not exist.')
+    template = re.compile(r'\b[a-zA-Z]{3,5}\b')
+    result_list = []
+    with open(filename,'r') as file:
+        list_of_strings = file.readlines()
+        for string in list_of_strings:
```

Comment:
The same comment on iteration through lines

Reply:
Ammended to 'for line in file:'